dist/leaflet.css

```diff
@@ -488,6 +488,9 @@ svg.leaflet-image-layer.leaflet-interactive path {
 	margin: 17px 0;
 	margin: 1.3em 0;
 	}
+.leaflet-popup-content:empty {

```

Comment:
The `:empty` pseudo-class is not desirable, https://github.com/Leaflet/Leaflet/issues/8131#issuecomment-1095181697:

> OTOH there could be really small content in popups, that this would fix. In which case `:empty` is not a good solution.